__tests__/index.test.js

```diff
@@ -374,4 +374,39 @@ describe('DELETE /api/comments/:comment_id', () => {
         expect(body.msg).toEqual(expected);
       });
   });
+});
+describe('GET /api/users', () => {
+  it('GET 200: should respond with an array of objects, each object should have the following properties: username, name, avatar_url', () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const { users } = body;
```

Comment:
If you feel comfortable with destructuring, you may like to try nested destructuring, which lets us do multiple rounds of destructuring on  one line https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8